File: app/src/main/java/com/fss/demo/util/ToastUtils.java
package com.fss.demo.util;

import android.content.Context;
import android.widget.Toast;

public class ToastUtils {

    private static Context mContext;

    public static void register(Context context) {
        mContext = context;
    }

    //显示Toast信息
    public static void showToast(String message) {
        if (mContext != null) {
            Toast.makeText(mContext, message, Toast.LENGTH_SHORT).show();
        }
    }

}

File: app/src/main/java/com/gongbo/fss/demo/util/ToastUtils.kt
package com.gongbo.fss.demo.util

import android.content.Context
import android.widget.Toast

object ToastUtils {

    private var mContext: Context? = null

    fun register(context: Context) {
        mContext = context
    }

    //显示Toast信息
    fun showToast(message: String) {
        if (mContext != null) {
            Toast.makeText(mContext, message, Toast.LENGTH_SHORT).show()
        }
    }

}
